frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -157,7 +158,7 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
           </FormGroup>
           <FormGroup
             fieldId="advanced-encryption-options"
-            label={t('ceph-storage-plugin~Connection settings')}
+            label={<AdvancedSubscription prefix={t('ceph-storage-plugin~Connection settings')} />}
```

Comment:
I think this should be in front of storage class encryption checkbox?

Reply:
Not according to the designs.

Reply:
@yuvalgalanti Can you check once I think it should be in the checkbox.

Reply:
Checked with @yuvalgalanti offline and he on moving to `storage class encryption` checkbox

Reply:
Moving next to `Storage Class Encryption`.